package.json

```diff
@@ -1,54 +1,54 @@
 {
-  "name": "filter-component",
-  "version": "0.0.0",
-  "description": "elastic.io filter component to filter incoming data based on arbitrary expression",
-  "homepage": "https://www.elastic.io",
-  "author": {
-    "name": "elastic.io GmbH",
-    "email": "info@elastic.io",
-    "url": "https://www.elastic.io"
-  },
-  "files": [
-    "lib"
-  ],
-  "main": "lib/index.js",
-  "keywords": [
-    "elasticio",
-    "filter",
-    "ipaas",
-    "elasticio-component"
-  ],
-  "dependencies": {
-    "bluebird": "^3.4.6",
-    "co": "^4.6.0",
-    "elasticio-node": "0.0.8",
-    "elasticio-sailor-nodejs": "2.1.0",
-    "moment": "^2.17.1",
-    "request": "^2.75.0",
-    "request-promise": "^4.1.1"
-  },
-  "scripts": {
-    "pretest": "node_modules/.bin/eslint lib spec Gruntfile.js --ext .json --ext .js --fix",
-    "test": "NODE_ENV=test grunt"
-  },
-  "devDependencies": {
-    "chai": "^3.5.0",
-    "eslint": "^3.1.1",
-    "eslint-config-xo-space": "^0.15.0",
-    "eslint-plugin-json": "^1.2.0",
-    "grunt": "^1.0.1",
-    "grunt-mocha": "^1.0.2",
-    "grunt-mocha-test": "^0.13.2",
-    "mocha": "^3.2.0",
-    "nock": "^9.0.2",
-    "sinon": "^1.17.7"
-  },
-  "eslintConfig": {
-    "extends": "xo-space",
-    "env": {
-      "mocha": true
-    }
-  },
-  "repository": "elasticio/filter-component",
-  "license": "Apache-2.0"
+    "name": "filter-component",
+    "version": "0.0.0",
+    "description": "elastic.io filter component to filter incoming data based on arbitrary expression",
+    "homepage": "https://www.elastic.io",
+    "author": {
+        "name": "elastic.io GmbH",
+        "email": "info@elastic.io",
+        "url": "https://www.elastic.io"
+    },
+    "files": [
+        "lib"
+    ],
+    "main": "lib/index.js",
+    "keywords": [
+        "elasticio",
+        "filter",
+        "ipaas",
+        "elasticio-component"
+    ],
+    "dependencies": {
+        "bluebird": "3.5.4",
+        "co": "4.6.0",
+        "elasticio-node": "0.0.9",
+        "elasticio-sailor-nodejs": "2.2.3",
+        "moment": "2.17.1",
+        "request": "2.75.0",
+        "request-promise": "4.2.4"
+    },
+    "scripts": {
+        "pretest": "node_modules/.bin/eslint lib spec Gruntfile.js --ext .json --ext .js --fix",
+        "test": "NODE_ENV=test grunt"
```

Comment:
If you change this section to:
```
{
  "pretest": "eslint lib/simpleJSONataFilter.js spec/simpleJSONataFilter.spec.js -fix",
  "test": "mocha spec/simpleJSONataFilter.spec.js"
}
```
then you will ignore the old tests and point to the new tests that you can write in `spec/simpleJSONataFilter.spec.js`

Reply:
Seems like the "-fix" in the pretest line needs to be a "--fix" otherwise it throws errors.